CMakeLists.txt

```diff
@@ -27,19 +27,27 @@ if (NOT CMAKE_BUILD_TYPE)
         FORCE)
 endif()
 
-set(AEON_VERSION_MAJOR 1)
-set(AEON_VERSION_MINOR 3)
-set(AEON_VERSION_PATCH 3)
+# Set directory where the custom finders live
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
+
+# Get actual aeon version number from sources
+include(AeonVersion)
 
-set(AEON_BUILD_NUMBER 0)
+AEON_GET_VERSION_LABEL()
```

Comment:
Why is this called before the next line below? Does the order matter? If it does, could you make it just one function call to prevent mistakes if someone reorders those calls? For example one function that calls both of those would be good enough in my opinion.

Reply:
I moved these 2 calls to one function in AeonVersion.cmake